sections/experiments/experiments-chapter.tex

```diff
@@ -281,13 +281,138 @@ \subsection{Performance Comparison (RQ2)}
+Having both context and side-information actually degrades performance slightly, compared to just having side-information as the input, and having just context as input also does not give much better results.
+To conclude from the study on the YelpNC dataset, the usage of context is not able to provide better recommendations in a non-context specific setting.
+Side-information, however, performs the best on this metric on this dataset, which suggest that side-information in our model is able to slightly improve results, compared to having no extra input.
+
+\begin{figure}
+    \begin{tikzpicture}
+        \begin{axis}[
+            xlabel=Epoch,
+            ylabel=NDCG@20,
+            height=8cm,
+            width=7cm,
+            grid=major,
+            legend style={at={(0.5,-0.20)},
+        anchor=north,legend columns=2}]
+            
+        \addplot plot coordinates{
+            (20,0.3749)
+            (40,0.3850)
+            (60,0.3854)
+            (80,0.3862)
+            (100,0.3836)
+            (120,0.3856)
+        };
 
+        \addplot plot coordinates {
+            (20,0.3798)
+            (40,0.3881)
+            (60,0.3900)
+            (80,0.3899)
+            (100,0.3862)
+            (120,0.3884)
+        };
 
-% For the YelpNC dataset in \Cref{tab:yelpnc-aggregated}, we see that CSGCN outperfoms the other methods in every metric, where the second best performing method on all metrics is the LightGCN method.
-% We see that CSGCN only gains small improvements ranging between 2-6\%, compared to the second best performing method.
-\todo[color=red]{Comparison of various aggregation methods}
+        \addplot plot coordinates {
+            (20,0.3748)
+            (40,0.3836)
+            (60,0.3816)
+            (80,0.3894)
+            (100,0.3841)
+            (120,0.3933)
+        };
+
+        \addplot plot coordinates {
+            (20,0.3749)
+            (40,0.3836)
+            (60,0.3893)
+            (80,0.3970)
+            (100,0.3912)
+            (120,0.3901)
+        };
+        \legend{Sideinfo + context,Sideinfo,Context,Nothing}
+        \end{axis}
+    \end{tikzpicture}
+    \caption{The performance on NDCG@20 of our model on the ML1M dataset with different types of input}
+    \label{fig:ablation_study_2}
+\end{figure}
+On \Cref{fig:ablation_study_2} the NDGC@20 results of ML1M is shown for our model with different types of input. 
+For this dataset, the model where neither the context or side-information is used in the input, is actually the best performing version.
+Side-information and context is not able to improve the performance of the model on ML1M in a non-context specific setting.
+Since both \Cref{fig:ablation_study_1,fig:ablation_study_2} show that the model is worse with context included, we can conclude that context does not increase the performance of our model in a non-context specific setting.
```

Comment:
```suggestion
Since both \Cref{fig:ablation_study_1,fig:ablation_study_2} show that the model is worse with context included, we can conclude that context does not increase the performance of CSGCN-ADJ in a non-context specific setting.
```